CollapsibleExample/ExampleSubclassFilterViewController.m

```diff
@@ -11,6 +11,7 @@
 @interface ExampleSubclassFilterViewController ()
 
 - (NSArray*)returnLabelArray;
```

Comment:
With these methods I would actually make readonly properties for these and remove the 'return' and 'Array' parts of the name. eg
```
@property (nonatomic, strong, readonly) NSArray *labels;
@property (nonatomic, strong, readonly) NSArray *interactions;
...
- (NSArray *)interactions {
NSArray* filterData = @[[[MHFilterLabel alloc] initLabelWithName:@"Comment Only" checked:NO interactionType:CRUCellViewInteractionCheckToggle], [[MHFilterLabel alloc] initLabelWithName:@"Spiritual Conversation" checked:NO interactionType:CRUCellViewInteractionCheckToggle],[[MHFilterLabel alloc] initLabelWithName:@"Personal Evangelism" checked:NO interactionType:CRUCellViewInteractionCheckToggle] , [[MHFilterLabel alloc] initLabelWithName:@"Personal Evangelism Decisions" checked:NO interactionType:CRUCellViewInteractionCheckToggle], [[MHFilterLabel alloc] initLabelWithName:@"Holy Spirit Presentation" checked:NO interactionType:CRUCellViewInteractionCheckToggle], [[MHFilterLabel alloc] initLabelWithName:@"Graduating on a Mission" checked:NO interactionType:CRUCellViewInteractionCheckToggle], [[MHFilterLabel alloc] initLabelWithName:@"Faculty on Mission" checked:NO interactionType:CRUCellViewInteractionCheckToggle]];
     return filterData;
}

```